packages/node/package.json

```diff
@@ -16,12 +16,12 @@
   "devDependencies": {
     "@counterfactual/apps": "0.1.1",
     "@counterfactual/firebase-server": "0.0.1",
+    "@counterfactual/postgresql-node-connector": "0.0.2",
```

Comment:
Seems odd that the Node has a dev dependency on a plugin to it.

Maybe the plugin should have a dev dependency on the node for testing purposes instead.

Reply:
That was my initial thought and is probably the better way to do it. Only reason I opted for this in the immediate future is because the main objective was to confirm the postgres implementation works as expected, but it's also because setting up proper e2e tests that use in the Node in the postgres package requires a lot of context